nomos-services/mempool/src/backend/mockpool.rs

```diff
@@ -67,7 +90,9 @@ where
         self.last_item_timestamp = SystemTime::now()
             .duration_since(UNIX_EPOCH)
             .unwrap()
-            .as_millis() as u64;
+            .as_millis()
+            .try_into()
+            .unwrap();
```

Comment:
Nitpik: Better to use `expect` with a hint message.

Reply:
Did it just here: https://github.com/logos-co/nomos-node/pull/1049/commits/568c170b6b36b5d30f38ce702b60e1c8026a1974